src/vcDBF.h

```diff
@@ -55,12 +55,14 @@ udResult vcDBF_GetFieldIndex(vcDBF *pDBF, const char *pFieldName, uint16_t *pInd
 uint16_t vcDBF_GetFieldCount(vcDBF *pDBF);
 uint32_t vcDBF_GetRecordCount(vcDBF *pDBF);
 
+uint16_t vcDBF_GetStringFieldIndex(vcDBF *pDBF);
```

Comment:
I still think this should be replaced with proper functions to get field information. We know the field count via `vcDBF_GetFieldCount`, so we should be able to query the field name, type and length based on the index.

Reply:
fixed.